lib/documents/schemas/asylum_support_decisions.json

```diff
@@ -160,6 +160,10 @@
           "label": "Bayati, C",
           "value": "bayati-c"
         },
+		{
+		  "label": "Beg, S",
+		  "value": "beg-s"
+		},
```

Comment:
Probably doesn't matter, but it looks like you've got a tabs vs. spaces inconsistency here.

Reply:
GAH!